src/planwise/component/facilities.clj

```diff
@@ -93,28 +96,36 @@
 (defn raster-isochrones! [service facility-id]
   (let [facilities-polygons-regions (select-facilities-polygons-regions-for-facility (get-db service) {:facility-id facility-id})]
     (doseq [{:keys [facility-polygon-id region-id] :as fpr} facilities-polygons-regions]
-      (let [population (-> service
-                          (:runner)
-                          (run-external :scripts "raster-isochrone" (str region-id) (str facility-polygon-id))
-                          (trim-newline)
-                          (trim)
-                          (Integer.))]
-        (set-facility-polygon-region-population!
-          (get-db service)
-          (assoc fpr :population population))))))
+      (try
+        (let [population (-> service
+                            (:runner)
+                            (run-external :scripts 60000 "raster-isochrone" (str region-id) (str facility-polygon-id))
+                            (trim-newline)
+                            (trim)
+                            (Integer.))]
```

Comment:
While this works fine, conceptually it mixes two different "pipelines": command construction and output processing. I think
```clojure
(-> (run-external (:runner service) :scripts 60000 ...)
    trim-newline
    trim
    Integer.)
```
has clearer intent (btw, the parens are optional when it's a single element)
Also, the `(-> x trim-newline trim Integer.)` pattern is used 3 times. Care to refactor in a util function? We probably need a general util namespace.